### ISSUE TITLE
[bitnami/tomcat] Unable to install tomcat if an initContainers is set within a yaml value file.

### ISSUE THREAD

**Issue by irenee25** [bitnami/charts#29158]
### Name and Version

bitnami/tomcat 11.2.18

### What architecture are you using?

amd64

### What steps will reproduce the bug?

Export default value within a file.
Just edit the content of initContainers.

For example : 
```
initContainers:
  - name: init-create-dir
    image: alpine
    imagePullPolicy: Always
```

### Are you using any custom parameters or values?

Yes, using a custom initContainers

```
initContainers:
  - name: init-create-dir
    image: alpine
    imagePullPolicy: Always
```

### What is the expected behavior?

The template must generate a template as below : 

```
      initContainers:
        - name: volume-permissions
          image: docker.io/bitnami/os-shell:11-debian-11-r94
          imagePullPolicy: "IfNotPresent"
          command:
            - /bin/bash
            - -ec
            - |
                chown -R 1001:1001 /bitnami/tomcat
          securityContext:
            runAsUser: 0
          resources:
            limits: {}
            requests: {}
          volumeMounts:
            - name: data
              mountPath: /bitnami/tomcat

        - image: alpine
          imagePullPolicy: Always
          name: init-create-dir

```

### What do you see instead?

      initContainers:
        - name: volume-permissions
          image: docker.io/bitnami/os-shell:11-debian-11-r94
          imagePullPolicy: "IfNotPresent"
          command:
            - /bin/bash
            - -ec
            - |
                chown -R 1001:1001 /bitnami/tomcat
          securityContext:
            runAsUser: 0
          resources:
            limits: {}
            requests: {}
          volumeMounts:
            - name: data
              mountPath: /bitnami/tomcat

      - image: alpine
        imagePullPolicy: Always
        name: init-create-dir

### Additional information

To fix the issue, edit the code of  tomcat/templates/_pod.tpl, near line 62.

Replace 

```
{{- if .Values.initContainers }}
{{ include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $)  }}
{{- end }}
```

by 

```
{{- if .Values.initContainers }}
{{ include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 2 }}
{{- end }}
```
( add indentation +2 )

Rendered template will be correct.

**Comment by javsalgar** [comment]
Hi!

Thank you so much for reporting! As you spotted it, would you like to submit a PR with the fix?

**Comment by irenee25** [comment]
> Hi!
> 
> Thank you so much for reporting! As you spotted it, would you like to submit a PR with the fix?

Honestly, i prefer that an expert will check, have a look and make changes. It is working locally on my side, but i have no idea of possible side effects on other K8S.
Moreover, sorry, i am not really fluent on git actions (forking, ... )
And, after checking git history, i could see that commit 806c0e82fd849a0490c4c6e62ae4f0bfdc2f8351 does the opposite of my proposal. I don't know the reason.

**Comment by migruiz4** [comment]
Thank you for reporting this issue @irenee25!

I have submitted this PR with your suggestion #29448 

The change at https://github.com/bitnami/charts/commit/806c0e82fd849a0490c4c6e62ae4f0bfdc2f8351 was correct, but later at some point, we modified the volumePermissions indent and forgot to set it back for initContainers, causing this issue. 

As I mentioned in the PR's 'Additional information' section, the indentation is not wrong by itself, both with and without initial whitespace are valid as long as they are used consistently, which is not the case when volumePermissions (indent 2) is used with initContainers (indent 0).
